utilities/simulator_cache/sim_cache.cc

```diff
@@ -50,6 +180,9 @@ class SimCacheImpl : public SimCache {
     } else {
       key_only_cache_->Release(h);
     }
+
+    cache_activity_logger_.ReportAdd(key, charge);
```

Comment:
Not related to this diff, but just a question: do we plan to always have SimCache enabled in MyRocks and other use cases, and turn on logging in run time? If that is the case, maybe we should have a switch to enable and disable the simulating part while it is running too. I think it is pretty cool if we can do that way.